.github/workflows/artifacts.yml

```
@@ -0,0 +1,77 @@
+name: Build artifacts
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  alpine_build:
+    name: Alpine Build
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker-compose run build
+    - run: docker-compose run Profiler.Alpine
+    - run: docker-compose run package.alpine
+    - run: mkdir artifacts
+    - run: cp -a deploy/linux/. artifacts/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: artifacts
+        path: artifacts
+
+  linux_build:
+    name: Linux Build
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker-compose run build
+    - run: docker-compose run Profiler
+    - run: docker-compose run package
+    - run: mkdir artifacts
+    - run: cp -a deploy/linux/. artifacts/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: artifacts
+        path: artifacts
```

consider (and test if you like it):

```suggestion
    - uses: actions/upload-artifact@v2
      with:
        name: artifacts
        path: deploy/linux
```

the same comment applies to other places